storybook/src/utils/AspectRatio/AspectRatio.stories.tsx

```diff
@@ -3,24 +3,25 @@
  * Copyright Gemeente Amsterdam
  */
 
+import { aspectRatioOptions } from '@amsterdam/design-system-react/src/common/types'
 import { Meta, StoryObj } from '@storybook/react'
 import { AspectRatio } from './AspectRatio'
 
 const meta = {
   title: 'Utilities/CSS/Aspect Ratio',
   component: AspectRatio,
   args: {
-    aspectRatio: 'x-wide',
+    aspectRatio: '16:9',
   },
   argTypes: {
     aspectRatio: {
       control: 'radio',
-      options: ['2x-wide', 'x-wide', 'wide', 'square', 'tall', 'x-tall'],
+      options: aspectRatioOptions,
     },
   },
   render: ({ aspectRatio }) => (
     <div className="ams-docs-column ams-docs-aspect-ratio">
-      <div className={`ams-docs-item ams-aspect-ratio--${aspectRatio}`}></div>
+      <div className={`ams-docs-item ams-aspect-ratio--${aspectRatio?.replace(':', '-')}`}></div>
```

Comment:
Even though this is a mock component, this could end up in a class of `ams-aspect-ratio--undefined`.

The same goes for the `Image` used in Image Slider. Can you replace that class with the prop?

Reply:
This mock component is only used in Storybook, connected to a radio control with a default value. That makes it impossible for it to be undefined, I think? 

It is an issue for Image in ImageSlider, but I think we should solve that in a different way.

Reply:
I've used `generateAspectRatioClass` here to handle this